net/udp-broadcast-relay-redux-openwrt/Makefile

```
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=udp-broadcast-relay-redux
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=$(PKG_SOURCE_VERSION)
+PKG_LICENSE:=GPL-2.0
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/udp-redux/udp-broadcast-relay-redux
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=671372938b55a186625a80516f86e8b9948c977a
```

Please add PKG_SOURCE_DATE, like this:
```suggestion
PKG_SOURCE_DATE:=2021-04-05
PKG_SOURCE_VERSION:=671372938b55a186625a80516f86e8b9948c977a
```

---

Will do!  My guess is that should always match the most recent commit date?

---

No, it should match the `PKG_SOURCE_VERSION` commit date.

---

Yeah, that's what I meant.  Got it!